src/rust/madara/sources/lermangas/res/source.json

```
@@ -0,0 +1,18 @@
+{
+	"info": {
+		"id": "pt-br.lermangas",
+		"lang": "pt-br",
+		"name": "Ler Mangas",
+		"version": 1,
+		"url": "https://lermangas.me",
+		"nsfw": 2
```

I think this has some but shouldn't be considered fully nsfw
```suggestion
		"nsfw": 1
```